atlite/resource.py

```diff
@@ -34,10 +34,11 @@
 def get_windturbineconfig(turbine):
     """Load the wind 'turbine' configuration.
 
-    The configuration can either be one from local storage, then 'turbine' is
+    The configuration can be one from local storage, then 'turbine' is
     considered part of the file base name '<turbine>.yaml' in config.windturbine_dir.
-    Alternatively the configuration can be downloaded from the Open Energy Database (OEDB),
+    Alternatively, the configuration can be downloaded from the Open Energy Database (OEDB),
     in which case 'turbine' is a dictionary used for selecting a turbine from the database.
+    Finally, turbine can also be a path to a local config file.
```

Comment:
We need to do some adjustements here, as `config.windturbine_dir` does not exist (very old legacy piece of doc which shouldn't be here).

Reply:
E.g.

The configuration can be either one of the configurations shipped with atlite, one loaded from a local path or retrieved from the Open Energy Database (OEDB):
If one of the names of the preshipped turbines from `atlite.resources.windturbines` is provided, that turbine configuration is loaded and returned.
If a `pathlib.Path` object is provided, this local path is read and the turbine configuration from this Path is parsed.
If a string starting with "oedb:<turbine name>" is provided, atlite will try to read the turbine configuration for this <turbine name> from OEDB.